docs/Producing effective SARIF.md

```diff
@@ -104,509 +104,3 @@ Each rule has a description that describes its purpose, followed by one or more
 Rules that describe violations of **SHALL**/**SHALL NOT** requirements of the [SARIF specification](https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/sarif-v2.1.0-os.html) have numbers between 1000 and 1999, and always have level `"error"`.
 
 ---
```

Comment:
This file will not survive; I have written better documentation that I will soon add to the SARIF Tutorials. For that matter, your new ValidationRules.md might end up there.

But for now, just to keep things neat and tidy, would you please add the following sentence to the end of this file:

```
For documentation on all existing SARIF validation rules, please see [ValidationRules.md](ValidationRules.md).
```